### PR TITLE
Update package, URL, and copyright from Kortforsyningen to OSGeo

### PR DESCRIPTION
**Pull request by willcohen**

### CODE REVIEW

**Comment by desruisseaux** [comment]
I did not tested yet, but at a first look it seems all right.

**Comment by kbevers** [comment]
The code parts looks okay but you can't just change the copyright retroactively. Not sure if OSGeo is the correct copyright holder from now on either. I'll have to look into that and get back with better feedback.

**Comment by desruisseaux** [comment]
Ah yes indeed, I was too focused on the code. Transfering the copyright may require Kortforsyningen to produce some legal paper, but doing such transfer is probably not needed. I think it is also fine to have code hosted at OSGeo with different copyright holders. It is the case of PROJ code itself if I remember right. It is also the standard practice at Apache Software Foundation for example.

**Comment by kbevers** [comment]
There' are two problems here. The first is that "Kortforsyningen" is listed as a copyright holder in some places - that should be "Agency for Data Supply and Efficiency" (Kortforsyningen is a brandname within the agency and can't as such be a copyright holder). This should be corrected.
The second problem is that the copyright holder is overwritten here. That should be dealt with by adding new copyright lines for the years after 2019. So we have something like

> Copyright (c) Agency for Data Supply and Efficiency, 2019-2021
> Copyright (c) Open Source Geospatial Foundation, 2021-20xx

Still not sure if OSGeo is the proper copyright holder but at least it's better than nothing. Individual developers might also want to put their name in I guess.

**Comment by willcohen** [comment]
Apologies. Will revise!

**Comment by kbevers** [comment]
> Apologies

No worries, it's not easy to get these things right. After having done the copyright and license review of PROJ my spidey-sense is always tingling when it comes to meddling with the legalese

**Comment by willcohen** [comment]
Apologies for losing track of this. Updated the statements to reflect a 2019-2021 and 2021 copyright, and force-pushed so that there's no incorrect labelling on an intermediate commit.

**Comment by desruisseaux** [comment]
No problem, thanks for the update. Just tried and I confirm that it builds and passes the tests at least on my machine (not yet upgraded to PROJ 8). Opened a few random files and it seems all right. On a very minor note, some files have a trailing space on the following lines:

```
Copyright © 2019-2021 Agency for Data Supply and Efficiency
```

**Comment by willcohen** [comment]
Good catch -- find/replace issue!

**Review by kbevers — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -7,16 +7,16 @@ geospatial coordinates from one coordinate reference system (CRS) to another.
...
-  * [Installation guide](https://kortforsyningen.github.io/PROJ-JNI/install.html)
-  * [PROJ binding Javadoc](https://kortforsyningen.github.io/PROJ-JNI/index.html)
+  * [Installation guide](https://osgeo.github.io/PROJ-JNI/install.html)
+  * [PROJ binding Javadoc](https://osgeo.github.io/PROJ-JNI/index.html)
   * [GeoAPI 3.0.1 Javadoc](https://www.geoapi.org/3.0/javadoc/index.html)
   * [Example](./example)
 
-**WARNING:** the `kortforsyningen` namespace in package name, module name, Maven artifact, _etc._
+**WARNING:** the `osgeo` namespace in package name, module name, Maven artifact, _etc._
 is temporary. The final name will be selected later.
```
[the remark this answers is not in the document]
Comment (willcohen) [reply README.md]: Lines removed with 8cac538!

**Review by kbevers — APPROVED** [review]

**Pull request merged**